runtime/near-vm-runner/src/wasmtime_runner.rs

```diff
@@ -79,6 +79,9 @@ pub mod wasmtime_runner {
                 Some(VMLogicError::InconsistentStateError(e)) => {
                     VMError::InconsistentStateError(e.clone())
                 }
+                Some(VMLogicError::EvmError(e)) => {
+                    VMError::FunctionCallError(FunctionCallError::EvmError(e.clone()))
```

Comment:
Somewhat inconsistent with above, either return error or assert in both cases. We usually return errors, I presume.

Reply:
Good catch.

Reply:
Good catch